frontend/src/app/AppRoutes.tsx

```diff
@@ -59,7 +61,10 @@ const AppRoutes: React.FC = () => {
 
         <Route path="/projects/*" element={<ProjectViewRoutes />} />
 
-        <Route path="/notebookController/*" element={<NotebookController />} />
+        {isJupyterEnabled && (
+          <Route path="/notebookController/*" element={<NotebookController />} />
```

Comment:
If we are hiding away `/notebookController`, why aren't we hiding the `workbenches` section in projects?

Reply:
The `/notebookController` is only the Jupyter tile on the `Enabled` page, it's nothing to do with the project workbenches, they are different features.

Reply:
mmmm so we don't have a way to hide workbenches right now in projects?

Reply:
I don't think we have a way to do that, we can only disable projects, but not the workbench section only.